### PR TITLE
Release/24.1

### PR DESCRIPTION
**Pull request by hlebkanonik**
# What Changed
- Deleted minio and switched to FS
- Added OpenSearch
- Added new envs for release 24.1

### CODE REVIEW

**Review by raikbitters — APPROVED** [review]

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ReportPortal  <https://reportportal.io>
+Copyright (C) 2016 EPAM Systems, Inc.
```
Comment (APiankouski) [inline NOTICE:2]: Do we need update year to 2024?

File: public/rp_repo_structure.png
Comment (APiankouski) [inline public/rp_repo_structure.png:1]: Если мы перешли на opensearch можно это в архитектуре отобразить

File: docker-compose.yml [hunk 2]
```diff
@@ -231,20 +161,48 @@ services:
...
-      - RP_AMQP_APIPASS=rabbitmq
-      - RP_BINARYSTORE_TYPE=minio
-      - RP_BINARYSTORE_MINIO_ENDPOINT=http://minio:9000
-      - RP_BINARYSTORE_MINIO_ACCESSKEY=minio
-      - RP_BINARYSTORE_MINIO_SECRETKEY=minio123
-      - LOGGING_LEVEL_ORG_HIBERNATE_SQL=info
-      - JAVA_OPTS=-Xmx1g -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp  -Dcom.sun.management.jmxremote.rmi.port=12349 -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.local.only=false  -Dcom.sun.management.jmxremote.port=9010 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Djava.rmi.server.hostname=0.0.0.0
+      ## Double entry moves test logs from PostgreSQL to Elastic-type engines
+      ## Ref: https://reportportal.io/blog/double-entry-in-5.7.2
+      RP_ELASTICSEARCHLOGMESSAGE_HOST: http://opensearch:9200
```
Comment (APiankouski) [inline docker-compose.yml:166]: in 24.1 the name of this var was changed to RP_ELASTICSEARCH_HOST

Comment (APiankouski) [reply docker-compose.yml:166]: also variables RP_ELASTICSEARCH_USERNAME and RP_ELASTICSEARCH_PASSWORD need to add

**Review by APiankouski — APPROVED** [review]

**Pull request merged**